### PR TITLE
Feature test for logout controller

### LINKED ISSUE #9: Add test for logout controleler
Write a mocha test for logout controller to return a successful response

### PR DESCRIPTION
**Pull request by sholabioku**
### What this PR does (required):
- This PR write a mocha test for the logout controller to return a successful response

### Any information needed to test this feature (required):
- Run npm test in the terminal

### CODE REVIEW

File: server/test/auth.test.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  describe('GET /auth/logout ', () => {
+    it('should return a successful response', async () => {
+      const res = await chai.request(app).get('/auth/logout');
+
+      res.should.have.status(200);
+      res.text.should.equal(message);
+    });
+  });
+});
```
Comment (sundayezeilo) [inline server/test/auth.test.js:19]: :+1:

**Review by sundayezeilo — COMMENTED** [review]
Looks good! Though there are changes in the other files due to your linters setup. Please do use the recommended linters config for all your projects. Check out these guides: 
- [VS Code](https://www.loom.com/share/bb8aaa6234ff4ce1b48306d16e268486)
- [Atom](https://www.loom.com/share/8025d3c6bd8343e4bee1a0cb61ee1698)
- [Sublime Text](https://www.loom.com/share/12ae38b15a774887924c94b56561861c)

**Review by rajivtitus — APPROVED** [review]
LGTM! Please address Sunday's comment :)

File: server/test/auth.test.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Mocha test for auth controller', () => {
+  const message = 'You have successfully logged out';
+
+  describe('GET /auth/logout ', () => {
+    it('should return a successful response', async () => {
+      const res = await chai.request(app).get('/auth/logout');
```
Comment (rajivtitus) [inline server/test/auth.test.js:13]: ust so we are aware, GET isn't the verb that makes the most sense for a /logout path! (see this [article](https://www.baeldung.com/logout-get-vs-post) for more details on why) For now though I think this is fine to leave as-is 👍🏼

The existing implementation may have thrown you off but this is the pitfall of developers building tests - we have to be careful not to be biased by our knowledge of the implementation and focus on specs!

**Pull request merged**